package.json

```diff
@@ -19,7 +19,7 @@
     "eslint-plugin-eslint-plugin": "^2.3.0",
     "eslint-plugin-node": "^10.0.0",
     "eslint-plugin-prettier": "^3.4.1",
-    "eslint-plugin-vue": "^6.2.2",
+    "eslint-plugin-vue": "^7.20.0",
```

Comment:
I would like to use v9 if possible.
If we use v9, use vue-eslint-parser v9 as well.

Reply:
v8 would be a breaking change regarding supported Node version (as it drops Node 12) and I think we should keep the supported Node versions with what ESLint itself supports (even though Node 12 is now EOL).

I was planning on upgrading to `eslint-plugin-vue` v8 together with `vue-eslint-parser` in a follow-up PR

Reply:
You are right. I didn't check the node version 😓